public/static/locales/en/planetcash.json

```diff
@@ -1,5 +1,6 @@
 {
   "title": "PlanetCash",
+  "partnerSignupInfo": "Our Planet Cash/Bulk Codes feature is a feature specifically for Plant-for-the-Planet partners. Feel free to contact us at <2>partner@plant-for-the-planet.org</2> if you are interested.",
```

Comment:
I wonder if as the key and text is the same, the resource will only to have been translated once or twice in LingoHub. 🤔

Reply:
I'm guessing twice? Since it's contained in different files..

Reply:
Then I would like to raise the question if it might be possible to only use one translation for this Trans component?

Reply:
I'm trying to keep the text for different features in separate files as far as possible. Even though there may be some duplication, I feel it keeps the code clearer, and I do not have to import another translation file for a single string.

We do maintain some common strings, but this text is only going to be used in these 2 places - so I think it may be ok to duplicate it.

Reply:
Using translation memory in LingoHub will probably compensate for this then.